app/js/components/Dashboard.jsx

```diff
@@ -9,49 +9,76 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 
-import openmrsLogo from '../../img/openmrs-with-title-small.png';
-import loadingGif from '../../img/loading.gif'
+export default () => {
+  const openmrsServer = window.location.pathname.split('/')[1];
+  const addonManagerLink = `http://localhost:8080/${openmrsServer}/owa/addonmanager/index.html`;
 
-export default class Dashboard extends React.Component {
-  render() {
-    const openmrsServer = window.location.pathname.split('/')[1];
-    return (
-      <div className="list-group home-list">
-        <Link to="#" className="list-group-item list-group-item-action flex-column align-items-start active">
-          <div className="d-flex w-100 justify-content-between">
```

Comment:
Is this also just another change in formatting?

Reply:
No, I changed most of the HTML to suit the new design. I uploaded the new design on the ticket.